scripts/install-rust.sh

```diff
@@ -8,11 +8,17 @@
 
 set -Ee
 
+. /etc/os-release
+
 RUSTUP_NEW_INSTALL=0
 
 # NOTE: rustup is used to allow multiple toolchain installations.
 if command -v rustup >/dev/null 2>&1; then
-    rustup self update
+    if [[ "$ID" =~ "opensuse" ]] || [[ "$ID_LIKE" =~ "opensuse" ]]; then
+      sudo zypper up rustup
+    else
```

Comment:
It is expected that rustup is installed from https://rustup.rs/, not by a distro package.

Reply:
This case can actually just be removed. It was initially done to ensure users had a version of rustup new enough to support TOML-based toolchain files. I've done it in #531 to preserve the context.

Reply:
Understood!